src/librustc/mir/interpret/mod.rs

```diff
@@ -351,12 +357,6 @@ impl<'s> AllocDecodingSession<'s> {
     }
 }
 
-impl fmt::Display for AllocId {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "{}", self.0)
-    }
-}
```

Comment:
So we had both a `Debug` and a `Display` for `AllocId` and they were quite far away from each other in the file and `Debug` was the one that did "prettier" printing...